requirements.txt

```diff
@@ -0,0 +1,2 @@
+avh-api
```

Comment:
I believe this should be placed somewhere else than top-level. 

Either scripts/tests/avh or name it as `scripts/setup/requirements.avh.txt`